persistance/src/main/java/cz/muni/fi/pv254/entity/User.java

```diff
@@ -82,9 +80,13 @@ public User() { }
 
     public void setIsAdmin(Boolean isAdmin) { this.isAdmin = isAdmin; }
 
-    public LegalStatusEnum getLegalStatus() { return legalStatus; }
+    public Set<Recommendation> getRecommendations() {
+        return recommendations;
```

Comment:
I would probably return unmodifiable collection

Reply:
We can change it later